src/components/leaderBoard/Components/BoostYourRankComponents.tsx

```diff
@@ -0,0 +1,72 @@
+import Image from "next/image";
+import LockImg from "../../../../public/LockImg.svg";
+import { boostYourRank } from "./Constants";
+
+export const BoostYourRankComponents = () => {
+  return (
+    <>
+      <div
+        style={{
```

Comment:
Please use tailwind styles for all these basis things.
Do it all places.
For reference. https://tailwindcss.com/docs/width